syntax = "proto3";

package sbai_geometry_protos;

// This represents a vector in free space.
//
// This is semantically different than a point.
// A vector is always anchored at the origin.
// When a transform is applied to a vector, only the rotational component is applied.

message Vector3 {
    double x = 1;
    double y = 2;
    double z = 3;
}
